source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -333,7 +334,20 @@ def _getColumnContentRaw(self, index):
 		return buffer.value if buffer else None
 
 	def _getColumnContent(self, column):
-		return self._getColumnContentRaw(self.parent._columnOrderArray[column - 1])
+		targetColumn = self.parent._columnOrderArray[column - 1]
+		try:
+			return self._getColumnContentRaw(targetColumn)
+		except ArgumentError:
+			# In case that a 64-bit pointer to the text had been  returned /rjh
+			disp = DisplayModelTextInfo(self, 'all')

```

Comment:
Instead of 'all', use textInfos.POSITION_ALL. However, personally I'd first try a different approach, see below.